backend/organization/models/project.py

```diff
@@ -0,0 +1,150 @@
+from django.db import models
+
+from organization.models import (Organization,)
+
+
+def project_image_path(instance, filename):
+    return "projects/{}/{}".format(instance.id, filename)
+
+
+class Project(models.Model):
+    name = models.CharField(
+        help_text="Points to project name",
+        verbose_name="Name",
+        max_length=1024
+    )
+
+    slug = models.CharField(
```

Comment:
On other tables this was called `url_slug` so I would do the same here for consistency